modules/fee-market/src/lib.rs

```diff
@@ -255,7 +254,6 @@ pub mod pallet {
 		/// Update locked collateral for enrolled relayer, only supporting lock more. (Update market
 		/// needed)
 		#[pallet::weight(<T as Config<I>>::WeightInfo::update_locked_collateral())]
-		#[transactional]
```

Comment:
why remove this?

Reply:
It's the default behavior in the latest substrate, See https://github.com/paritytech/substrate/pull/11546